theseus/src/api/logs.rs

```diff
@@ -151,7 +229,71 @@ pub async fn delete_logs_by_datetime(
 
     let state = State::get().await?;
     let logs_folder = state.directories.profile_logs_dir(&profile_path).await?;
-    let path = logs_folder.join(datetime_string);
+    let path = logs_folder.join(filename);
     io::remove_dir_all(&path).await?;
     Ok(())
 }
+
+#[tracing::instrument]
+pub async fn get_latest_log_cursor(
+    profile_path: ProfilePathId,
+    mut cursor: u64, // 0 to start at beginning of file
+) -> crate::Result<LatestLogCursor> {
```

Comment:
what is this func for?

Reply:
Gets latest.log from a certain point (cursor) to the end and updates that cursor to mark the end of the file. Lets us get the updates to latest.log since the last time we checked (ie: not have to reload latest.log every tick to get a live log)